packages/ubuntu_bootstrap/lib/installer/installation_step.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:dartx/dartx.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:ubuntu_bootstrap/pages.dart';
+import 'package:ubuntu_provision/ubuntu_provision.dart';
+import 'package:ubuntu_wizard/ubuntu_wizard.dart';
+
+enum InstallationStep {
```

Comment:
This enum was previously only used for the progress indicator - this generalization nicely simplifies the route definitions :+1:.
However, there are some small issues with the indicator now - the second step is skipped and the number of steps changes:

[Screencast from 2024-01-02 14-36-32.webm](https://github.com/canonical/ubuntu-desktop-provision/assets/113362648/b2e9e06d-a81d-4879-8271-995c015a1e85)

Reply:
How was that handled before when the second step was skipped? Where did the length come from? :thinking:

Reply:
Aah right, because that step wasn't defined as a discrete step before.